lib/image-stream/gzip.js

```diff
@@ -38,37 +35,34 @@ const ISIZE_LENGTH = 4;
  * by reading its `ISIZE`. The specification clarifies that this
  * value is just an estimation.
```

Comment:
Done! In any case, passing a `read()` function that uses HTTP Ranges, for example, allows you to avoid downloading the whole file